pos/.gitignore

```diff
@@ -0,0 +1,22 @@
+# dependencies
+/node_modules
```

Comment:
You probably mean `/node_modules/` to explicitely exclude the folder just like you did with `/.next/`